symfony/twig-bundle/5.4/templates/base.html.twig

```diff
@@ -14,6 +14,14 @@
         {% endblock %}
     </head>
     <body>
+        {% for type, messages in app.flashes %}
+            {% for message in messages %}
+                <div class="flash flash-{{ type }}">
```

Comment:
I know that within Symfony we use the "flash message" terminology, but in the front-end niche they are more commonly called "alert messages" or "alerts":

* Bootstrap alerts: https://getbootstrap.com/docs/5.0/components/alerts/
* TailwindCSS alerts: https://tailwindui.com/components/application-ui/feedback/alerts
* Segment Evergreen UI Kit alerts: https://evergreen.segment.com/components/alert
* Ant Design UI Kit alerts: https://ant.design/components/alert/

So, maybe this code could be:

```twig
<div class="alert alert-{{ type }}" role="alert">
```

The `role` attribute is optional, but improves accessibility a bit. Cheers!

Reply:
Good call, made this change.